components/gateway/cmd/main.go

```diff
@@ -139,7 +140,7 @@ func initAuditLogs(done chan bool) (AuditogService, error) {
 			msgFactory = auditlog.NewMessageFactory(oauthCfg.User, oauthCfg.Tenant, uuidSvc, timeSvc)
 		}
 	default:
-		return nil, errors.New(fmt.Sprintf("Invalid Auditlog Auth mode: %s", cfg.AuthMode))
+		return nil, errors.New(fmt.Sprintf("Invalid auditlog auth mode: %s", cfg.AuthMode))
```

Comment:
You can use `fmt.Errorf(...)` and also go errors should start with lowercase and logs with uppercase (valid for the whole change)